test/unit/engine_sequencer.js

```diff
@@ -153,9 +153,19 @@ test('stepToProcedure', t => {
     t.strictEquals(th.peekStack(), expectedBlock);
     s.stepToProcedure(th, 'faceCode');
     t.strictEquals(th.peekStack(), expectedBlock);
-    s.stepToProcedure(th, 'faceCode');
-    th.target.blocks.getBlock(th.stack[th.stack.length - 4]).mutation.proccode = 'othercode';
+
+    th.target.blocks.createBlock({
+        id: 'internalId',
+        opcode: 'procedures_defnoreturn_internal',
```

Comment:
Not a block type.  Did you mean `procedures_callnoreturn_internal`?

Reply:
Ah good catch. It isn't actually used in the code (only the outer block opcode is used) but good to have it there to make it clear what is going on.